### PR TITLE
Fix Composer scripts

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see comments below.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,9 +40,9 @@
         "phpstan/phpstan": "^1.4"
     },
     "scripts": {
-        "phpcs": "vendor/bin/phpcs",
+        "phpcs": "phpcs",
         "phan": "export PHAN_DISABLE_XDEBUG_WARN=1; vendor/bin/phan",
-        "phpstan": "vendor/bin/phpstan",
-        "lint": ["phpcs", "phan", "phpstan"]
+        "phpstan": "phpstan",
+        "lint": ["@phpcs", "@phan", "@phpstan"]
```
Comment (szepeviktor) [inline composer.json:46]: `@` references another script.

File: composer.json [hunk 2]
```diff
@@ -40,9 +40,9 @@
         "phpstan/phpstan": "^1.4"
     },
     "scripts": {
-        "phpcs": "vendor/bin/phpcs",
+        "phpcs": "phpcs",
         "phan": "export PHAN_DISABLE_XDEBUG_WARN=1; vendor/bin/phan",
-        "phpstan": "vendor/bin/phpstan",
```
Comment (szepeviktor) [inline composer.json:45]: Composer find `bin` scripts. Also on Windows!

**Review by marekdedic — APPROVED** [review]

**Comment by marekdedic** [comment]
Thanks!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**